wizytowka-master-TOM/stylesheet.css

```diff
@@ -0,0 +1,87 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-size: 100%;
+  font-family: Lato
+}
```

Comment:
tutaj jedna linia przerwy między klasami w cssie

Reply:
właściwie to w całym pliku powinieneś dodać jedną linię przerwy pomiędzy sąsiedującymi klasami